Scripts/InstallPyPackages.py

```diff
@@ -67,12 +67,13 @@ def install(*packages):
     upgradePip()
 
     installFromGit(owner='ikibalin', repo='cryspy', branch='transition-to-version-0.2')
+    installFromGit(owner='easyDiffraction', repo='easyInterface', branch='master') # Until master branch is uploaded to PIP
 
     install(
         #'cryspy==0.1.13',
         'scipy==1.4.1',
         'numpy==1.18.1',
-        'easyInterface>=0.0.6',
+        #'easyInterface>=0.0.6',
```

Comment:
Is this supposed to be commented out? Will the master branch of easyInterface be sufficient?

Reply:
Currently, min/max implementation is already pushed into master branch, but PIP package is not yet updated. So, we need this for the moment.